### PR TITLE
Total Requests by Response + type and fix caps

### PR DESCRIPTION
**Pull request by sarathecoubian**
- Overview
Inbound - Total requests by type // Inbound - Total requests by response type
Outbound - Total requests by type // Outbound - Total requests by response type

- Inbound
Total inbound requests by type // Total inbound requests by response type

- Outbound
Total outbound requests by type // Total outbound requests by response type

Also fixed capitalization of chart names.

### CODE REVIEW

File: modules/ROOT/pages/app-dashboards.adoc [hunk 1]
```diff
@@ -55,17 +55,19 @@ Built-in dashboards include various pages to view different types of charts:
...
-* Maximum response time
-* Total failed inbound requests
-* Slow request count by endpoint: Total calls in which response time is greater than 1000 ms
+* Total Requests by Endpoint
+* Response Time graphs (99, 90, 75, and 50 percentile inbound): Measured in milliseconds (ms)
+* Maximum Response Time
+* Total Failed Inbound Requests
+* Slow Request Count by Endpoint: Total calls in which response time is greater than 1000 ms
+
+// I don't see Maximum Response Time or Total Failed Inbound Requests in the UI - do those exist?
```
Comment (sarathecoubian) [inline modules/ROOT/pages/app-dashboards.adoc]: @pgaliana Please advise here. Might not be looking in the right place?

File: modules/ROOT/pages/app-dashboards.adoc [hunk 2]
```diff
@@ -119,25 +121,25 @@ If you don't see the *Flows* tab, verify that you are using Unified Agent and ha
...
-* Failed inbound requests: Response time by endpoint
-* Total failed inbound requests grouped by endpoint
-* Failed outbound requests: Response time by HTTP endpoint (only HTTP endpoints are shown)
-* Total failed outbound requests grouped by HTTP endpoint (only HTTP endpoints are shown)
+* Total Failed Inbound Requests
+* Total Failed Outbound Requests
+* All Inbound Requests grouped by response type
+* All Outbound Requests grouped by response type
+* Failed Inbound Requests: Response time by endpoint
+* Total Failed Inbound Requests Grouped by Endpoint
```
Comment (dhirsch1) [inline modules/ROOT/pages/app-dashboards.adoc]: For consistency with other entries, "Grouped by Endpoint"  should be lowercase unless it is part of the name.

File: modules/ROOT/pages/app-dashboards.adoc [hunk 3]
```diff
@@ -55,17 +55,19 @@ Built-in dashboards include various pages to view different types of charts:
...
-* Total requests by endpoint
-* Response time graphs (99, 90, 75, and 50 percentile inbound): Measured in milliseconds (ms)
-* Maximum response time
-* Total failed inbound requests
-* Slow request count by endpoint: Total calls in which response time is greater than 1000 ms
+* Total Requests by Endpoint
+* Response Time graphs (99, 90, 75, and 50 percentile inbound): Measured in milliseconds (ms)
+* Maximum Response Time
+* Total Failed Inbound Requests
+* Slow Request Count by Endpoint: Total calls in which response time is greater than 1000 ms
```
Comment (dhirsch1) [inline modules/ROOT/pages/app-dashboards.adoc]: "the response time"?

Comment (sarathecoubian) [reply modules/ROOT/pages/app-dashboards.adoc]:
```suggestion
* Slow Request Count by Endpoint: Total calls in which the response time is greater than 1000 ms
```

File: modules/ROOT/pages/app-dashboards.adoc (hunk 2 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/app-dashboards.adoc]:
```suggestion
* Total Failed Inbound Requests grouped by endpoint
```

**Pull request merged**